app/api_topology.go

```diff
@@ -106,7 +106,7 @@ func handleWebsocket(
 			log.Errorf("Error generating report: %v", err)
 			return
 		}
-		renderer, decorator, err := topologyRegistry.rendererForTopology(topologyID, r.Form, report)
+		renderer, decorator, err := topologyRegistry.RendererForTopology(topologyID, r.Form, report)
```

Comment:
OK, this justifies the `Registry` being public, although you could create a wrapper like `AddContainerFilters`